contracts/test/IWETH.sol

```diff
@@ -1,7 +1,8 @@
 // SPDX-License-Identifier: GPL-3.0
 
-import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+pragma solidity ^0.8.0;
```

Comment:
I think we should rewrite history so the pragma is added within the same commit that added the `IWETH` interface.

Reply:
done